default/AI/DiplomaticCorp.py

```diff
@@ -4,63 +4,65 @@
 
 import freeOrionAIInterface as fo  # pylint: disable=import-error
 import FreeOrionAI as foAI
-import AIstate
-from freeorion_tools import UserString, UserStringList, chat_on_error, print_error, chat_human, ppstring
+from freeorion_tools import UserStringList, chat_on_error
+
 
 def handle_pregame_chat(sender_id, message_txt):
     possible_acknowledgments = UserStringList("AI_PREGAME_ACKNOWLEDGEMENTS__LIST")
     acknowledgement = random.choice(possible_acknowledgments)
-    print ("Acknowledging pregame chat with initial message (from %d choices): '%s'" %
-          (len(possible_acknowledgments), acknowledgement))
+    print "Acknowledging pregame chat with initial message (from %d choices): '%s'" % (
+        len(possible_acknowledgments), acknowledgement)
     fo.sendChatMessage(fo.empirePlayerID(sender_id), acknowledgement)
 
-class DiplomaticCorp(object):
 
+class DiplomaticCorp(object):
     @chat_on_error
-    def handle_diplomatic_message(self, message):  # pylint: disable=invalid-name
+    def handle_diplomatic_message(self, message):
         """Handle a diplomatic message update from the server,
         such as if another player declares war, accepts peace, or cancels a proposed peace treaty.
         :param message:
         :return:
         """
-        print "Received diplomatic %s message from empire %s to empire %s" % (message.type, message.sender, message.recipient)
-        print "my empire id: %s" % fo.empireID()
+        print "Received diplomatic %s message from %s to %s." % (
+            message.type, fo.getEmpire(message.sender),
+            'me' if message.recipient == fo.empireID() else fo.getEmpire(message.recipient))
         if message.recipient != fo.empireID():
             return
         reply_sender = message.recipient
```

Comment:
regarding reply_recipient, message.sender, etc.,  consolidating terms makes plenty of sense, but now that the method is not only about making a reply, I think it better to just stick with message.sender, message.recipient terms.  We can  drop the use of reply_recipient and reply_sender.